rust/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "proto-build"
+version = "0.1.0"
+authors = ["OKP4"]
+edition = "2021"
+publish = false
+
+
+[dependencies]
+prost = "0.10"
+prost-build = "0.10"
```

Comment:
Can we put `prost-build` and `tonic-build` in build dependencies instead ?

Reply:
Since the build script is never build by rust making such changes will not work